src/main/java/view_post/interface_adapters/ViewPostViewModel.java

```diff
@@ -10,15 +8,13 @@
  */
 public class ViewPostViewModel {
     private final PropertyChangeSupport observable;
-    private final ViewPostView view;
 
     /**
      * Initializes ViewPostViewModel.
      * @param view A ViewPostView object
      */
-    public ViewPostViewModel(ViewPostView view){
```

Comment:
Maybe I'm just out of the loop, but why is the view being removed from this class?